### ISSUE TITLE
Allow specifying which folders should be checked by the syncthing sync check

### ISSUE THREAD

**Issue by surskitt** [spruceUI/spruceOS#246]
I am currently sharing two folders with my miyoo using syncthing. One containing my saves and a much larger one for the roms.

The syncthing sync check is often held up waiting on this large roms folder to finish scanning. Since syncthing is stopped while a game is running a new scan will happen before and after playing a game which ends up causing a wait on both startup and shutdown.

The saves folder is the one I would want to be in sync before shutting down the system or starting up a new game. It is small and the syncing and scanning does not take too long. It is not as important to keep the roms folder up to date so it could be ignored by the sync check.

**Comment by BaseInfinity** [comment]
Hey there!

So it looks like a quick workaround you can do is visit your Syncthing GUI on your A30 and `Pause` the `Roms` directory for syncing.

The Syncthing Sync Check will now ignore that directory when doing a Sync Check. You can always `Resume` it when you enable new Roms.

We are currently discussing a long term solution here and trying to be thoughtful about use cases here

**Comment by surskitt** [comment]
I've already done exactly that! Wasn't completely sure it was doing what I wanted or needed, but it's good to have a confirmation.
